docs/modules/ROOT/pages/references/crds.adoc

```diff
@@ -958,6 +976,30 @@ VSHNPostgreSQLParameters are the configurable fields of a VSHNPostgreSQL.
 | *`monitoring`* __xref:{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlmonitoring[$$VSHNPostgreSQLMonitoring$$]__ | Monitoring contains settings to control monitoring.
 | *`encryption`* __xref:{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlencryption[$$VSHNPostgreSQLEncryption$$]__ | Encryption contains settings to control the storage encryption of an instance.
 | *`updateStrategy`* __xref:{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlupdatestrategy[$$VSHNPostgreSQLUpdateStrategy$$]__ | UpdateStrategy indicates when updates to the instance spec will be applied.
+| *`instances`* __integer__ | Instances configures the number of PostgreSQL instances for the cluster. Each instance contains one Postgres server. Out of all of the Postgres servers, one is elected as the primary, the rest remain as read-only replicas.
+| *`replication`* __xref:{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlreplicationstrategy[$$VSHNPostgreSQLReplicationStrategy$$]__ | This section allows to configure Postgres replication mode and HA roles groups. 
+ The main replication group is implicit and contains the total number of instances less the sum of all instances in other replication groups.
+|===
+
+
+[id="{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlreplicationstrategy"]
+=== VSHNPostgreSQLReplicationStrategy 
+
+
+
+.Appears In:
+****
+- xref:{anchor_prefix}-github-com-vshn-appcat-apis-vshn-v1-vshnpostgresqlparameters[$$VSHNPostgreSQLParameters$$]
+****
+
+[cols="25a,75a", options="header"]
+|===
+| Field | Description
+| *`mode`* __string__ | Mode defines the replication mode applied to the whole cluster. Possible values are: "async"(default), "sync", and "strict-sync" 
```

Comment:
According to the stackgres docs, there are two additional modes `sync-all` and `strict-sync-all`. Why are they not listed here?

Reply:
Yeah, because they don't exist yet in `1.4` that we currently use. https://stackgres.io/doc/1.4/reference/crd/sgcluster/#replication

Reply:
Ah right. In that case all is good